csrc/transformer/cublas_wrappers.cu

```diff
@@ -88,7 +88,9 @@ int cublas_gemm_ex(cublasHandle_t handle,
                                          CUDA_R_32F,
 #endif					
                                          m,
-#ifdef __HIP_PLATFORM_AMD__ 
+#if defined(__HIP_PLATFORM_AMD__) && HIPBLAS_V2
```

Comment:
@rraminen Are these and other similar instances meant to check if HIPBLAS_V2 is defined or if it is set to 1? If it's the former, the check should be `defined(HIPBLAS_V2)`?